data/json/monsters/insect_spider.json

```diff
@@ -2057,8 +2057,30 @@
     "death_function": { "message": "The %s's body leaks acid.", "effect": { "id": "death_acid", "hit_self": true } },
     "harvest": "arachnid_acid",
     "dissect": "dissect_insect_sample_single",
+    "upgrades": { "half_life": 42, "into": "mon_ant_acid_soldier_mega" },
     "flags": [ "ACIDPROOF", "SHORTACIDTRAIL", "CLIMBS", "HEARS", "POISON", "SEES", "SMELLS", "PATH_AVOID_FIRE", "PATH_AVOID_FALL" ]
   },
+  {
+    "id": "mon_ant_acid_soldier_mega",
+    "type": "MONSTER",
+    "name": { "str": "super soldier acid ant" },
+    "description": "A cow-sized, woolly brown ant.  Corrosive liquid seeping from her huge mandibles.  Given the relative strength of ants to their body size, you don't doubt that this one could shove cars around or rip through solid walls.",
+    "copy-from": "mon_ant_acid_soldier",
+    "proportional": { "hp": 10, "speed": 0.25, "dodge": 0.5, "vision_day": 2 },
+    "volume": "625 L",
+    "weight": "815 kg",
+    "melee_dice": 2,
+    "armor_elec": 4,
+    "bleed_rate": 60,
+    "special_attacks": [
+      { "id": "smash", "throw_strength": 48, "cooldown": 30 },
+      [ "BIO_OP_TAKEDOWN", 20 ],
+      [ "RANGED_PULL", 20 ],
+      [ "GRAB", 5 ]
```

Comment:
Consider `GRAB_DRAG` instead of these two, that's usually a more interesting (and very anty) attack.
A `grab_strength` of 2-3 might make sense for them, too.
Not a fan of using the hardcoded biooperator attacks for other enemies, so I'd go with a JSON special.

Also, where's the acid? This will overwrite the base mob's attacks